src/frontend/components/UI/UninstallModal/index.tsx

```diff
@@ -91,7 +89,6 @@ const UninstallModal: React.FC<UninstallModalProps> = function ({
       navigate('/')
     }
     storage.removeItem(appName)
-    refreshLibrary({ fullRefresh: true, checkForUpdates: false })
```

Comment:
This was not needed. When uninstalling a game, the backend sends a `refreshLibrary` message to the frontend triggering this same call (we were running this twice on uninstall)